frontend/app/styles/encryptable-form.scss

```diff
@@ -0,0 +1,60 @@
+.encryptable-form-name {
+  border-bottom-color: rgb(222, 226, 230);
+  border-bottom-style: solid;
+  border-bottom-width: 1px;
+}
+
+.add-field {
```

Comment:
please use more bootstrap wherever possible... the less custom classes the better. give yourself quite a bit of time to work through your css and replace it with bootstrap wherever possible

Reply:
Bootstrap was implemented in encryptable/forms.hbs